pkg/kf/commands/apps/integration_test.go

```diff
@@ -83,6 +83,40 @@ func TestIntegration_Push(t *testing.T) {
 	})
 }
 
+// TestIntegration_Push_manifest pushes the manifest app, using a manifest.yml
+// file. The app is identical to the echo app, and this fact is used to also
+// test manifest file environment variables. It finally deletes the app.
+func TestIntegration_Push_manifest(t *testing.T) {
+	t.Parallel()
+	RunKfTest(t, func(ctx context.Context, t *testing.T, kf *Kf) {
+		appName := "manifest-app"
```

Comment:
I'm glad you noticed because I was hoping to get help on a solution for this issue.

https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html
`To push a single app rather than all of the apps described in the manifest, provide the desired app name by running cf push my-app`

So if you supply an app name (which our tests do) it assumes we want to do a single-app deploy of a possible multi-app manifest file.

Here I have `manifest-app` and not `manifest-app-Time.now()` for that reason.
We could update the manifest file as part of the test? What do you think?

Reply:
How hard would it be to manipulate the manifest?

Reply:
I'd hesitate to update the test data because then we might be modifying it in unexpected ways which could make the tests invalid. Instead, what do you think about a flag (possibly hidden) that overrides the pushed app's name?

Reply:
I think I would rather just stick with `manifest-app` over a hidden flag. That said, I would hope generating a YAML manifest would be easy enough.

Reply:
++ to generating the manifest; I'd just prefer not to edit existing ones which assumes our parser is complete and correct.

Reply:
I now copy and edit the manifest file int he sample app, to update the name to be unique per test